src/renderer/dx/DxRenderer.cpp

```diff
@@ -341,11 +341,24 @@ HRESULT DxEngine::_SetupTerminalEffects()
 // - <none>
 void DxEngine::_ComputePixelShaderSettings() noexcept
 {
-    // Retro scan lines alternate every pixel row at 100% scaling.
-    _pixelShaderSettings.ScaledScanLinePeriod = _scale * 1.0f;
+    if (_retroTerminalEffects && _d3dDeviceContext && _pixelShaderSettingsBuffer)
+    {
+        // Retro scan lines alternate evolution
+        _pixelShaderSettings.Downscale = _scale * 2.0f;
+
+        // Set the display resolution
+        _pixelShaderSettings.Width = 1.0f*_displaySizePixels.width<UINT>();
+        _pixelShaderSettings.Height = 1.0f*_displaySizePixels.height<UINT>();
 
-    // Gaussian distribution sigma used for blurring.
-    _pixelShaderSettings.ScaledGaussianSigma = _scale * 2.0f;
+        // Set the background
+        _pixelShaderSettings.Background = _backgroundColor;
+
+        try

```

Comment:
I moved the resource update here as this is now needed to be updated on background change as well DPI change